packages/create-next-app/helpers/examples.ts

```diff
@@ -102,7 +102,9 @@ export async function downloadAndExtractRepo(
     strip: filePath ? filePath.split('/').length + 1 : 1,
     filter: (p) =>
       p.startsWith(
-        `${name}-${branch.replace(/\//g, '-')}${filePath ? `/${filePath}` : ''}`
+        `${name}-${branch.replace(/\//g, '-')}${
+          filePath ? `/${filePath}/` : ''
```

Comment:
Not sure when we get the case where `filePath == false`, but maybe there should a slash in the other branch too?

```suggestion
filePath ? `/${filePath}/` : '/'
```